tpm/constants.go

```diff
@@ -162,6 +164,34 @@ const (
 	authPrivUseOnly byte = 0x03
 )
 
+// Locality Selection
+
+const (
+	locFour  byte = 0x16
+	locThree byte = 0x8
+	locTwo   byte = 0x4
+	locOne   byte = 0x2
+	locZero  byte = 0x1
+)
+
+// nvPremissions and Operations
+
+const (
+	nvPerPPWrite    uint32 = 0x00000001
+	nvPerOwnerWrite uint32 = 0x00000002
+	nvPerAuthWrite  uint32 = 0x00000004
+	nvPerWriteAll   uint32 = 0x00000800
+	//REALLY REALLY BAD!!!!!!!!!!!!!!!!
```

Comment:
Please revise this comment to make clear what the danger is for this value.

If the value is so dangerous that it is not provided by the library (since it's commented out), then I'd propose not giving it the form of code at all. For example, these lines could read something like the following (filling in the missing information with the right values):

```go
// WARNING: The value 0x1000 is defined in the spec as <whatever name it has in the spec>,
// but it is not included directly in this code because <reason that it is dangerous>.
// Please do not use that value unless you know what you're doing.
```

Then you have to at least convert this comment into code before using the dangerous value.